assets/js/common/SuseManagerConfig/CertifiedUploadDate.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { format } from 'date-fns';
+import { EOS_LOCK_OUTLINED } from 'eos-icons-react';
+
+function CertificateUploadDate({ date }) {
+  if (!date) {
+    return '-';
+  }
+
+  return (
+    <div className="flex flex-row items-center">
+      <EOS_LOCK_OUTLINED className="mr-3" size="25" />
+
+      <div>
+        <div>Certificate Uploaded</div>
+        <div className="text-xs">{format(date, 'iii MMM dd, y')}</div>
```

Comment:
I just like a shorter date as per example but it might be too early to make a call on what is right or wrong with date structure. I think as long as there is consistency with how we display dates across Trento then we should be fine.